code_search/kubeflow/components/nms.libsonnet

```diff
@@ -2,7 +2,49 @@ local baseParams = std.extVar("__ksonnet/params").components["nmslib"];
 
 {
 
-  nmsContainer(params, env):: {
+  indexCreator(params, env):: {
```

Comment:
Do we need a libsonnet file?
The pattern we are generally following is to avoid complex ksonnet and just put the full spec in the jsonnet file so its for people to do advanced customizaiton.